tests/unit/containers/tutor-home-page/add-photo-step/AddPhotoStep.spec.jsx

```diff
@@ -0,0 +1,39 @@
+import AddPhotoStep from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep'
+import { renderWithProviders } from '~tests/test-utils'
+import { screen } from '@testing-library/react'
+import { vi } from 'vitest'
```

Comment:
Import libraries first 
```suggestion
import { screen } from '@testing-library/react'
import { vi } from 'vitest'
import AddPhotoStep from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep'
import { renderWithProviders } from '~tests/test-utils'
```

Reply:
Thanks! I've corrected it.